app/transport/http/server.go

```diff
@@ -10,37 +10,41 @@ import (
 	"github.com/gorilla/mux"
 )
 
-const timeoutMsg = "Connection timeout"
+const (
+	timeoutMsg  = "Connection timeout"
+	handlerTime = 3
```

Comment:
Let's call it `handlerTimeoutSeconds` and analogically for other constants. To bring more clarity to these names.

Reply:
Usually, classic handler timeout is 30 seconds, by the way. Any specific reasons for such a strict requirement here?

Reply:
fixed